docs/mockups/background.html

```diff
@@ -76,8 +76,8 @@
       </span>
      </button>
      <a class="text-nowrap" href="home.html" style="text-decoration: none">
-      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" height="50" src="../www/img/logo_transparent.png" title="VAPALS-ELCAP"/>
-      <img alt="VAPALS-ELCAP Logo" height="50" src="../www/img/logo_name.png" title="VAPALS-ELCAP"/>
+      <img alt="VAPALS-ELCAP Logo" class="hidden-xs" src="../www/img/logo_transparent.png" style="height:50px;" title="VAPALS-ELCAP"/>
```

Comment:
Why did you move the height from the tag attribute to CSS? Won't this cause the page to potentially reflow after it processes the CSS? Just curious why you changed it. Was it required for 508 compliance?

Reply:
It was not for 508, but was something the IDE complained about, which really just complained that the image size was not the same size as it's rendered size. I will back this out and actually create an image that is the appropriate size, so no scaling will be needed. That will make the page load faster too.

Reply:
Sounds good.